pkg/api/types.go

```diff
@@ -455,6 +455,9 @@ type PersistentVolumeSpec struct {
 	// means that this volume does not belong to any StorageClass.
 	// +optional
 	StorageClassName string
+	// A list of mount options. Not validated--mount will simply fail if one is invalid.
```

Comment:
Can we provide an example here, so people don't think this means `[ "-oro", "-o soft", ]`